lib/hammer_cli/ca_cert_manager.rb

```diff
@@ -0,0 +1,71 @@
+require 'fileutils'
+
+module HammerCLI
+  class CACertManager
+    attr_reader :ca_path
+
+    def initialize(ca_path)
+      @ca_path = ca_path
+    end
+
+    def add_ca_cert(uri)
+      raw_cert = get_ca_cert(uri)
+      cert_file = cert_file_name(uri)
+      store_ca_cert(raw_cert, cert_file)
+    end
+
+    def get_ca_cert(uri)
```

Comment:
Not a big thing but I would move the logic around fetching certs to a standalone class for the sake of separating responsibilities (you could then avoid stubbing `get_ca_cert` in tests). But I'm not going to block the PR if you don't like it.